### PR TITLE
CHM-251-Upgrade scbd-common to Vue3

### PR DESCRIPTION
**Pull request by tahmeedshah**
upgrade packages.
convert scbd-header, scbd-footer and cop-count-down components to vue-3

How to run/test:
step.1 : npm install
step2. run script '"build": "rm -rf ./dist && rollup -c"
step.3: run script 'dev'

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,28 +44,31 @@
     "cleanup:hard": "rm -rf ./dist && rm -rf ./node_modules && rm -rf ./package-lock.json"
   },
   "dependencies": {
-    "vue": "^2.7.10"
+    "moment": "^2.30.1",
+    "scss": "^0.2.4",
```
Comment (blaisefonseca) [inline package.json]: this should be dev dependency

File: package.json [hunk 2]
```diff
@@ -44,28 +44,31 @@
     "cleanup:hard": "rm -rf ./dist && rm -rf ./node_modules && rm -rf ./package-lock.json"
   },
   "dependencies": {
-    "vue": "^2.7.10"
+    "moment": "^2.30.1",
```
Comment (blaisefonseca) [inline package.json]: we do not refer this lib, if it is used should be under peerDependency

File: package.json [hunk 3]
```diff
@@ -44,28 +44,31 @@
...
+    "scss": "^0.2.4",
+    "vue": "^3.4.15",
+    "vue-router": "^4.2.5"
```
Comment (blaisefonseca) [inline package.json]: we are not building a app, please remove router and the test app

File: package.json [hunk 4]
```diff
@@ -44,28 +44,31 @@
...
   },
   "dependencies": {
-    "vue": "^2.7.10"
+    "vue": "^3.4.15"
   },
   "peerDependencies": {
     "axios": "^1.3.3",
     "lodash": "^4.17.21",
     "luxon": "^3.2.1",
+    "moment": "^2.30.1",
```
Comment (blaisefonseca) [inline package.json]: where is moment used?

Comment (tahmeedshah) [reply package.json]: at articles-accordion.vue component so for

Comment (blaisefonseca) [reply package.json]: lets use luxon, as its already referenced in the lib, https://github.com/scbd/scbd-common/blob/master/src/services/filters/datetime.js

**Review by sbilodeau — APPROVED** [review]

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**